src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -130,7 +130,7 @@ def _get_credentials(cli_ctx,
     :param str repository: Repository for which the access token is requested
     :param str permission: The requested permission on the repository, '*' or 'pull'
     """
-    registry, _ = get_registry_by_name(cli_ctx, registry_name, resource_group_name)
+    registry, resource_group_name = get_registry_by_name(cli_ctx, registry_name, resource_group_name)
```

Comment:
Does the resource group name get transformed somehow?  Why do you need it to return from the method when you are supplying it as a parameter?

Reply:
The incoming `resource_group_name` can be empty but the returned must have a value. The method `get_registry_by_name` has a different logic depending on whether `resource_group_name` is `None`.